spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe 'User posts', type: :feature do
+  describe 'Show Page' do
+    before(:each) do
+      @user1 = User.create(name: 'Usom', photo: 'https://source.unsplash.com/user/c_v_r/100x100', bio: 'Usom bio')
+      @user2 = User.create(name: 'Burger', photo: 'https://i.picsum.photos/id/216/200/300.jpg?hmac', bio: 'Burger bio')
+      @post1 = Post.create(title: 'Post 1', text: 'Post content goes here', author_id: @user1.id)
+      @comment1 = Comment.create(post_id: @post1.id, text: 'First comment goes here', author_id: @user1.id)
+      @comment2 = Comment.create(post_id: @post1.id, text: 'Second comment goes here', author_id: @user2.id)
+      @comment3 = Comment.create(post_id: @post1.id, text: 'Third comment goes here', author_id: @user2.id)
+      @like1 = Like.create(author_id: @user1.id, post_id: @post1.id)
+      @like2 = Like.create(author_id: @user1.id, post_id: @post1.id)
+      @like3 = Like.create(author_id: @user2.id, post_id: @post1.id)
+
+      visit user_posts_path(@user1)
+    end
+
+    it 'Should have profile picture content of all other users' do
+      expect(page).to have_css("img[src*='/images/default.jpg']")
+    end
```

Comment:
- [ ] Please follow up on my recommendation on how to test a profile picture of a user in a previous comment. Please make sure to test the `photo: value` in the `src: value`.